custom_components/tuya_local/devices/jiahong_et72w_thermostat.yaml

```diff
@@ -60,6 +60,7 @@ entities:
         type: boolean
         mapping:
           - dps_val: false
+            default: true
```

Comment:
default: true is only used by the siren entity to select a siren tone to use when none is provided in the action parameters and there is no current value being reported by the device.

To define a read-only default value when there is no current value being reported in general, you should just remove the `dps_val: false`. If this is required, it is probably required for the other uses (except the select entity where it needs to be read-write).

Reply:
Thanks, I've removed the `default` configs. I was attempting to resolve and issue where one of my devices was having wifi connectivity issues and as a result it was showing error messages and not defining the temperature measurement unit C/F at all.

Correct fix is to sort out my wifi here.